sli/slistartup.cc

```diff
@@ -214,53 +214,55 @@ SLIStartup::SLIStartup( int argc, char** argv )
     StringDatum* sd = new StringDatum( argv[ i ] );
     args_array.push_back( Token( sd ) );
 
-    if ( *sd == "-d" or *sd == "--debug" )
+    std::string myarg = argv[i];
```

Comment:
Our general policy is to declare `const` for anything that does not change for clarity. We also require whitespace inside parentheses and brackets:
```suggestion
    const std::string myarg = argv[ i ];
```